File: aspnet-core/src/ManageApartments.Core/Domain/Enums/InvoiceType.cs
using System.ComponentModel;

namespace ManageApartments.Domain.Enums

{
    public enum InvoiceType
    {
        [Description("Aidat")]
        Fee = 0,
        [Description("Kira")]
        Rent = 1,
        [Description("Elektrik")]
        Electric = 2,
        [Description("Diğer")]
        Other = 3,


    }
}

File: angular/src/app/service/enum/InvoiceType.ts
import { BaseEnum } from "@shared/components/data-grid/enums/base-enum";

export enum InvoiceType {

    Aidat = 0,
    Kira = 1,
    Elektrik = 2,
    Diğer = 3,
}

export const InvoiceTypeEnum: Array<BaseEnum> = [
    { id: 0, displayName: "Aidat" },
    { id: 1, displayName: "Kira" },
    { id: 2, displayName: "Elektrik" },
    { id: 3, displayName: "Diğer" }

];
